File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/OpenAiFileList.kt
package com.sdkwork.ai

data class OpenAiFileList(
    val data: List<OpenAiFileObject>? = null,
    val object: String? = null,
    val firstId: String? = null,
    val lastId: String? = null,
    val hasMore: Boolean? = null
)

File: sdkwork-ai-sdk-go/types/open_ai_file_list.go
package types

// OpenAI标准文件列表对象
type OpenAiFileList struct {
	Data []OpenAiFileObject `json:"data"`
	Object string `json:"object"`
	FirstId string `json:"first_id"`
	LastId string `json:"last_id"`
	HasMore bool `json:"has_more"`
}
